docs/reference/ml/apis/evaluateresources.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-evaluate-dfanalytics-resources]]
+=== {dfanalytics-cap} evaluation resources
+
```

Comment:
This page is different than most of the other pages in the "Definitions" section, since it seems to be defining the input (request body) properties for the evaluate DF analytics API, rather than the output (response body) properties. In many other cases, the input and output is similar (i.e. input to create job matches output from get jobs so the "job resources" applies to both).  That doesn't seem to be the case here, though.

I think we should either (a) extend the evaluation resources page to also describe the response objects, or (b) move the configuration objects into the API reference page and only cover the response objects in the resources page.

If I've explained this poorly or misunderstood the goal of this page, just let me know!

Reply:
The resources page contains both the request body and the response body parameters.

Some params are overlapping, for example `auc_roc`, `precison`, `recall`, `confusion_matrix` can be also the part of the request body as well as of the response body. `tp`, `fp`, `tn`, `fn` could be only the part of the response body. As far as I see, all the response objects are covered here.